src/index.js

```diff
@@ -33,7 +33,7 @@ const forcedServices =
   process.env.NODE_ENV === 'development' || process.env.REACT_APP_IS_DEMO_ENABLED
     ? {
         dopplerLegacyClient: new HardcodedDopplerLegacyClient(),
-        dopplerSitesClient: new HardcodedDopplerSitesClient(),
+        //dopplerSitesClient: new HardcodedDopplerSitesClient(),
```

Comment:
Here you need to remove the comment for it to work in demo and dev right?

Reply:
there is wrong! thanks for the advice